sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/history/OfferOutcome.java

```diff
@@ -0,0 +1,39 @@
+package com.mesosphere.sdk.offer.history;
+
+import org.apache.mesos.Protos;
+
+public class OfferOutcome {
+    private final long timestamp;
+    private final String podInstanceName;
+    private final boolean pass;
+    private final Protos.Offer offer;
+    private final String outcomeDetails;
+
+    public OfferOutcome(String podInstanceName, boolean pass, Protos.Offer offer, String outcomeDetails) {
+        this.timestamp = System.currentTimeMillis();
+        this.podInstanceName = podInstanceName;
+        this.pass = pass;
```

Comment:
Would it make sense to rather store the `failedOutcomeCount`?

Reply:
I went with the boolean for now.